schemas/FeatureManagement.Dotnet.v1.0.0.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "A.NET Feature Management Configuration",
+  "required": [],
+  "patternProperties": {
+    "^[^:]*$": {
+      "description": "Declares a feature flag.",
+      "anyOf": [
+        {
+          "type": "boolean",
+          "title": "On/Off Feature Flag",
+          "description": "A feature flag that always returns the same value."
+        },
+        {
+          "type": "object",
+          "title": "Conditional Feature Flag",
+          "description": "A feature flag which value is dynamic based on a set of feature filters",
+          "required": [
+            "EnabledFor"
+          ],
+          "properties": {
+            "RequirementType": {
+              "type": "string",
+              "title": "Requirement Type",
+              "description": "Determines whether any or all registered feature filters must be enabled for the feature to be considered enabled.",
+              "enum": [
```

Comment:
This is not ideal because enums enforce case- and this value is case insensitive. We could just make it a string with any value (and supply "example"s of All and Any). Or use regex- but that workaround isn't great. 

Alternatively- we leave it like this or change the enums to lower case- then update our examples & tests to adhere to the case (currently they dont)

Reply:
We should declare the schema the proper way.

Reply:
> We should declare the schema the proper way, with specific enum value.

We should update our tests and examples to adhere to it then.

Reply:
Completely agree. What ones did you find?

Reply:
FeatureUsesFiltersWithDuplicatedAlias and AllFeatureFilter.

All valid now! Updated in https://github.com/microsoft/FeatureManagement-Dotnet/pull/372